lib/krane/kubernetes_resource.rb

```diff
@@ -228,6 +228,11 @@ def type
       @type || self.class.kind
     end
 
+    def group
+      grouping, *version = @definition.dig("apiVersion").split("/")
```

Comment:
Do we need the `*` prefix for `version`?

Reply:
True, just version will work the same